jenkins-scripts/tools/outdated-job-runner.groovy

```diff
@@ -0,0 +1,68 @@
+import hudson.model.*;
```

Comment:
The code mixed here CamelCase with var_undescore_notation, it would be great to go with one or the other. Probably the first one since the Jenkins API is using that.